Lesson_07/homework_07.py

```diff
@@ -0,0 +1,163 @@
+# task 1
+""" The task is to print a multiplication table by a given number, but
+only up to the maximum value for the product - 25.
+The code is almost ready, it is necessary to find errors and correct/complement."""
+
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while True:
+        result = number * multiplier
+
+        # Stop the loop if the result exceeds 25
+        if  result > 25:
+            break
+
+        print(str(number) + "x" + str(multiplier) + "=" + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+
+print("1. Table of multiplication by a given number: ")
+(multiplication_table(3))
+print("-" * 100)
+# Should print:
+# 3x1=3
+# 3x2=6
+# 3x3=9
+# 3x4=12
+# 3x5=15
+
+# task 2
+"""  Write a function that calculates the sum of two numbers."""
+
+def calculating_function(num1, num2):
+
+    return num1 + num2
+
+print(f"2. Sum of two numbers: {(calculating_function(12, 2))}")
```

Comment:
No need of extra parentheses
```suggestion
print(f"2. Sum of two numbers: {calculating_function(12, 2)}")
```

Reply:
also everywhere else in the code